src/guppy/Packet.hpp

```diff
@@ -2,7 +2,7 @@
 
 typedef struct __attribute__((__packed__)) rf_down_t
 {
-    unsigned pad : 2;
+    unsigned pad : 8;
```

Comment:
this should just be 0. if we have an even number of bytes, we don't need a pad.
If it dosen't like 0 then just comment it out.

Reply:
Yeah it didn't like 0 pad. I'll comment it out